### ISSUE TITLE
Different configs for environments

### ISSUE THREAD

**Issue by Shiriaev** [ngraf/slack-codeception-extension#9]
Hello!

Is it possible to use different configs for environments? For example, enable notifications only for specific environment, not for all?

Thanks

**Comment by ngraf** [comment]
Yes, I think so.

Look at the the example at https://codeception.com/docs/07-AdvancedUsage#Current-values module how to achieve this.

In this case you can try:
```
env:
    development:
         # extension not enabled
    production:
         extensions:
            enabled:
               - Codeception\Extension\SlackExtension
    
```

But what I personally prefer is the configuration via command line. I always have the extension inactive in yaml configuration. Then I add `--ext SlackExtension` to my `codecept run --ext SlackExtension` to activate it. In local mode I don't add it, in my remote CI/CD pipelines it is added as parameter in the pipeline.
If this was helpful, then please let me know.

**Comment by Shiriaev** [comment]
Only one variant is working.
codeception.yml:

```
extensions:
  config:
    Codeception\Extension\SlackExtension:
      messagePrefix: '*Smoke-Test*'
      webhook: https://hooks.slack.com/services/...
      strategy: always
      extended: true
      extendedMaxLength: 150
      channel: '#any-channel'
```

And then 
`
php codecept --debug run api --ext SlackExtension
`

variant with 

```
env:
    development:
         # extension not enabled
    production:
         extensions:
            enabled:
               - Codeception\Extension\SlackExtension
```
Not working.
It would be great if you could set different prefixes for different environments, for example

``` 
messagePrefix: '*DEV Smoke-Test*'
messagePrefix: '*STAGE Smoke-Test*'
messagePrefix: 'PROD *Smoke-Test*'
```

**Comment by ngraf** [comment]
Take a look into https://codeception.com/docs/06-ModulesAndHelpers#Dynamic-Configuration-With-Parameters . 

I create custom `messagePrefix` myself as well. I use different `.env` files for each environment.
They are loaded in by adding this to your `codeception.yaml`:
````
params:
    - .env
````

The configuration for the SlackExtension's `messagePrefix` looks like this:
```
       messagePrefix: '%ENV% *Smoke-Test*'
```

My `.env.prod` file looks like this:
```
ENV=PROD
```

1) Before the test run I copy the appropiate .env file to `.env`, f.e. `cp .env.prod .env`
2) I start my test run as usual with `php codecept run --ext SlackExtension` and the message sent to Slack will be 'PROD *Smoke-Test*'.

**tl;dr**
Try to find a solution with the possibilites Codeception offers. The documentation is great. I am unsure  how to improve the extension itself to help you with your problem.

**Comment by ngraf** [comment]
I close those issue now. Issue should be solved with existing Codeception framework features not by extending the extension.
